### DISCUSSION TITLE
sudo so-start all fails for some reason.

### DISCUSSION THREAD

**Discussion by tanc7** [Security-Onion-Solutions/securityonion#9405] [Unsupported Versions]
![image](https://user-images.githubusercontent.com/26513830/207984922-6915b2ac-74a6-42ed-bbfa-55ab2492a07b.png)

Came back home noticing that none of the web GUI services are running. Running sudo so-status beforehand shows me this.

![image](https://user-images.githubusercontent.com/26513830/207985001-caac2a6f-4398-4a17-a138-b4a63471874d.png)

Checking this path returns empty directories

![image](https://user-images.githubusercontent.com/26513830/207986588-518d3973-3da9-47fa-ac2a-764df2cf12d3.png)

I think I found a bug in the latest release of SecurityOnion. For some reason the DHCP Client wouldn't automatically get a IP address. I had to follow this guide here https://superuser.com/questions/578293/linux-centos-how-to-permanent-inititate-dhcp-client-any-reboot

I had to ensure that the line **BOOTPROTO=dhcp** is present and I added two more lines

**ONBOOT=yes
HWADDR=62:BF:52:XX:XX:XX**

**For the management interface configuration on /etc/sysconfig/network-scripts/ifcfg-eth0**

**Comment by dougburks** [comment]
Please note that we generally recommend using a static IP address instead of DHCP:
https://docs.securityonion.net/en/2.3/configuration.html
